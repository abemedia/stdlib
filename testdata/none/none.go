package none

import (
	"none/lo"
)

func _(a, b string) {
	lo.CoalesceOrEmpty(a, b)
}
