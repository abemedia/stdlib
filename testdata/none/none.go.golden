package none

import (
	"expnone/lo"
)

func _(a, b string) {
	lo.CoalesceOrEmpty(a, b)
}
